src/bio/bio_buffer.c

```diff
@@ -1010,8 +1030,14 @@ bio_iod_copy(struct bio_desc *biod, d_sg_list_t *sgls, unsigned int nr_sgl)
 
 	arg.ca_sgls = sgls;
 	arg.ca_sgl_cnt = nr_sgl;
-
-	return iterate_biov(biod, copy_one, &arg);
+	arg.ca_zc_fetch = zc_fetch;
+	
+	int rc = iterate_biov(biod, copy_one, &arg);
+	d_iov_t *iov;
+	iov = arg.ca_sgls->sg_iovs;
+	D_DEBUG(DB_IO, "buf %s", (char *)iov->iov_buf);
+	return rc;
+	
```

Comment:
(style)  trailing whitespace

Reply:
removed the message to keep the code clean.